### DISCUSSION TITLE
parameter naming in distributions

### DISCUSSION THREAD

**Discussion by cburgard** [hep-statistics-standard/hep-statistics-standard#28] [Standard]
Hey everyone!

I'm going through past comments I have received. One from @kratsg addresses the lack of consistency in the naming of our distributions, namely:

- gaussian_dist uses `mean` and `sigma`
- lognormal_dist uses `mu` and `sigma`

The suggestion is to use either "mu/sigma" or "mean/stddev"

What do people think?

**Comment by jonas-eschle** [comment]
I agree with the comment, to use either one or the other and not mix both (std instead of stddev?)
I would prefer "mu", "sigma" slightly over "mean", "std", but no strong preference here.
